src/iterators/linear.rs

```diff
@@ -6,15 +6,15 @@ use iterators::{HistogramIterator, PickyIterator};
 pub struct Iter<'a, T: 'a + Counter> {
     hist: &'a Histogram<T>,
 
-    valueUnitsPerBucket: i64,
-    currentStepHighestValueReportingLevel: i64,
-    currentStepLowestValueReportingLevel: i64,
+    valueUnitsPerBucket: u64,
```

Comment:
Unrelated to this change, but there's a mix of camelCase and snake_case. Do you want me to make them all snake_case (as is the norm in rust-land) or leave them as is? I don't care strongly about it (I actually prefer camelCase visually, but I prefer to go with ecosystem norms even more than that), so if it's something you'd been meaning to get around to, I'll fix it, but otherwise I'll leave it.